### PR TITLE
Several asteroids spawn

### PR DESCRIPTION
**Pull request by Miho42**
More asteroids spawn after time defined by variable **ASTEROIDS_TIMER**

### CODE REVIEW

File: my_game.py [hunk 1]
```diff
@@ -329,6 +329,15 @@ def on_update(self, delta_time):
         # Update the player shots
         self.player_shot_list.update()
 
+        # Time between asteroid spawn count down
+        self.asteroids_timer -= 1 * delta_time
```
Comment (tobiasbp) [inline my_game.py:333]: What do you get from multiplying by one?

File: my_game.py [hunk 2]
```diff
@@ -28,6 +28,7 @@
 PLAYER_MAX_SPEED = 7
 UFO_CHANGE_DIR_TIME_MAX = 10
 UFO_CHANGE_DIR_TIME_MIN = 2
+ASTEROIDS_TIMER = 5
```
Comment (tobiasbp) [inline my_game.py:31]: Let's call this ASTEROIDS_TIMER_SECONDS. Also, put in a comment which explains what it does.

**Review by tobiasbp — CHANGES_REQUESTED** [review]

File: my_game.py (hunk 1 above)
Comment (Miho42) [reply my_game.py:333]: What i'm doing should be very clear to everyone.

but I can fix fix

**Review by tobiasbp — APPROVED** [review]
Looks good... But, maybe there is a set amount of Asteroids in a level?

**Pull request merged**